allensdk/brain_observatory/behavior/behavior_ophys_session.py

```diff
@@ -305,6 +306,21 @@ def segmentation_mask_image(self) -> Image:
     def segmentation_mask_image(self, value):
         self._segmentation_mask_image = value
 
+    @property
+    def eye_tracking_data(self) -> pd.DataFrame:
+        """A dataframe containing DeepLabCut eye tracking ellipse fit
+        parameters for eye, pupil, and corneal reflection.
+
+        :rtype: pandas.DataFrame
+        """
```

Comment:
I think this docstring should be expanded quite a bit to explain the following:

* all columns starting with 'pupil_' represent ellipse fit parameters relating to the pupil
* all columns starting with 'eye_' represent ellipse fit parameters relating to the eyelid
* all columns staring with 'cr_' represent ellipse fit parameters relating to the corneal reflection, which is caused by an infrared LED positioned near the eye tracking camera.
* positions are in units of pixels.
* areas are in units of pixels^2
* all values are in the coordinate space of the eye tracking camera, NOT the coordinate space of the stimulus display (i.e., this is not gaze location), with 0,0 being the upper-left corner of the image.
* the 'likely_blink' column is True for any row where pupil fit failed OR eye fit failed OR an outlier fit was identified
* all ellipse fits are derived from tracking points that were output by DeepLabCut model that was trained on hand-annotated data from a subset of imaging sessions on the optical physiology rigs. The raw tracking points are not available [but should the be?]